environment/bashrc/.bashrc

```diff
@@ -78,18 +76,18 @@ case ${-} in
     esac
     ;; # end case 'interactive'
 
-  ##---------------------------------------------------------------------------##
-  ## ENVIRONMENTS for non interactive sessions
-  ##---------------------------------------------------------------------------##
+  #---------------------------------------------------------------------------#
+  # ENVIRONMENTS for non interactive sessions
+  #---------------------------------------------------------------------------#
```

Comment:
Minor: Should these be updated to 100 columns? :)

Reply:
Fixed by 7b03549.